js/src/block-editor/components/test/fields.js

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import '@testing-library/jest-dom/extend-expect';
-import React from 'react';
+import * as React from 'react';
```

Comment:
Not related to this PR, but it [looks like](https://epicreact.dev/importing-react-through-the-ages) it's better with `*`